framework/addons/utils.py

```diff
@@ -53,7 +53,15 @@ def read_capabilities(filename):
 here = os.path.abspath(here)
 
 lookup = TemplateLookup(
-    directories=[os.path.join(here, 'templates')]
+    directories=[os.path.join(here, 'templates')],
+    default_filters=[
+        'unicode',  # default filter; must set explicitly when overriding
+        'temp_ampersand_fixer',  # FIXME: Temporary workaround for data stored in wrong format in DB. Unescape it before it gets re-escaped by Markupsafe.
```

Comment:
Cool. There will be an effort in the near future to tag all TODO and FIXME items with Jira tickets for bi-directional traceability. This keeps us from having to rely on memory or continued survival of a small number of people or continuous re-investigation. Having them tagged up front in the code will keep us from having to double that effort.